cmd/security-secrets-setup/README.md

```diff
@@ -60,11 +60,11 @@ To run the **import** subcommand in linux system, change directory to *cmd/secur
 mkdir -p /tmp/pki/cache && export PKI_CACHE=/tmp/pki/cache && sudo -E ./security-secrets-setup import
 ```
 
-There are two cases can happen depends on the status of cache directory:
+There are two cases can happen depending on the status of cache directory:
 
 1. Cache dir */tmp/pki/cache* is empty:
 
-    In this case, one should expect to see the execution is blocking and waiting for TLS assets to be drop into cache directory before they can be deployed into directory */run/edgex/secrets/*.
+    In this case, import just errors out: the TLS assets is required to be in the cache directory before they can be deployed into directory */run/edgex/secrets/*.
```

Comment:
The intent here was that if the assets are being copied from an image into a mounted docker volume, that the container would wait for this to finish.  Does docker guarantee that this will happen before the container starts executing?